u2f-gae-demo/src/com/google/u2f/gaedemo/storage/TokenStorageData.java

```diff
@@ -119,8 +113,7 @@ public boolean equals(Object obj) {
     TokenStorageData that = (TokenStorageData) obj;
     return (this.enrollmentTime == that.enrollmentTime)
         && SecurityKeyData.containSameTransports(this.transports, that.transports)
-        && (this.counter == that.counter)
-        && Arrays.equals(this.keyHandle, that.keyHandle)
+        && (this.counter == that.counter) && Arrays.equals(this.keyHandle, that.keyHandle)
```

Comment:
This is the official eclipse formatting style.  I wanted to go with it so we don't have to format only parts of the file automatically.  But won't push.  Can revert this. if you'd like.